File: backend/src/main/java/org/jdk/project/dto/statistics/PassengerRecordDto.java
package org.jdk.project.dto.statistics;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class PassengerRecordDto {
  private Long id;
  private String roomCode;
  private String deptName;
  private String passengerName;
  private String flightNo;
  private String flightDate;
  private String cardProvider;
  private String cardNo;
  private String accessType;
  private String accessStatus;
  private String checkInAt;
  private String checkOutAt;
  private String regionName;
  private String cabin;
  private String seatNo;
  private String starLevel;
  private String originalImageUrl;
}

File: backend/src/main/kotlin/org/jdk/project/dto/statistics/PassengerRecordDto.kt
package org.jdk.project.dto.statistics

data class PassengerRecordDto(
    var id: Long? = null,
    var roomCode: String? = null,
    var deptName: String? = null,
    var passengerName: String? = null,
    var flightNo: String? = null,
    var flightDate: String? = null,
    var cardProvider: String? = null,
    var cardNo: String? = null,
    var accessType: String? = null,
    var accessStatus: String? = null,
    var checkInAt: String? = null,
    var checkOutAt: String? = null,
    var regionName: String? = null,
    var cabin: String? = null,
    var seatNo: String? = null,
    var starLevel: String? = null,
    var originalImageUrl: String? = null,
)
